src/udev/udev-builtin-net_setup_link.c

```diff
@@ -35,7 +35,11 @@ static int builtin_net_setup_link(UdevEvent *event, int argc, char **argv) {
 
                 /* Set ID_NET_NAME= with the current interface name. */
                 const char *value;
-                if (sd_device_get_sysname(dev, &value) >= 0)
+                if (naming_scheme_has(NAMING_USE_INTERFACE_PROPERTY))
+                        r = device_get_ifname(dev, &value);
+                else
+                        r = sd_device_get_sysname(dev, &value);
```

Comment:
doesn't matter, but i#d have used a ternary op for this

Reply:
That makes the line longer... I prefer the current form.